packages/abstract-sdk/src/cw_helpers/cw_messages.rs

```diff
@@ -0,0 +1,74 @@
+use crate::{AbstractSdkError, AbstractSdkResult};
+
+use abstract_core::proxy::ExecuteMsg;
+use cosmwasm_std::{wasm_execute, CosmosMsg, WasmMsg};
+
+#[derive(Debug, PartialEq)]
+pub enum AbstractMessage {
+    ProxyMessage { msgs: Vec<CosmosMsg>, proxy: String },
+    Raw(CosmosMsg),
```

Comment:
Could you explain why we'd add a raw variant? I don't see a usage of it.

Reply:
> Could you explain why we'd add a raw variant? I don't see a usage of it.

I wanted the api functions with messages to always return the same type of structure